cclib/parser/gamessparser.py

```diff
@@ -689,14 +689,26 @@ def extract(self, inputfile, line):
                 except ValueError:
                     pass
                 else:
-                    values.append([float(line.split()[self.scf_valcol])])
+                    # if there were 100 iterations or more, the first part of the line
+                    # will look like 10099, 101100, 102101, etc., with no spaces between
+                    # the numbers. We can check if this is the case by seeing if the first
+                    # number on the line exceeds 10000.
+                    gt_100_its = int(line.split()[0]) > 10000
```

Comment:
Can you store the result of `line.split()` so it can be used here and reused again on lines 698 and 702?

Reply:
Done!